### DISCUSSION TITLE
Get a .png from mod's resource folder and draw it to screen

### DISCUSSION THREAD

**Discussion by StupidRepo** [FabricMC/fabric-api#3479] [Mod Dev Support] [answered]
### Description

I know I can use `DrawContext.drawSprite` to draw a sprite, but how do I get a .png as a Sprite which I can then draw?

**Comment by Linguardium** [comment]
you probably want DrawContext.drawTexture instead

**Reply by StupidRepo** [reply, marked as answer]
Yup just realised that right now. Silly me didn't read the other functions.
